stock_picking_invoicing/README.rst

```diff
@@ -25,6 +25,7 @@ Contributors
 * Leonardo Pistone <leonardo.pistone@agilebg.com>
 * Daniel Sadamo <daniel.sadamo@kmee.com.br>
 * Alex Comba <alex.comba@agilebg.com>
+* Florent THOMAS <florent.thomas@mind-and-go.com>
```

Comment:
@flotho l  Thanks for your contribution to the OCA. 
Please follow the principles: 
* Contribution guidelines: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md
* Can you please update to the newest OCA README template:
https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst
* We recommend to include a screenshot as well to help new comers (nice to have)
* Please use simple headers in all py files following OCA templates
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3